daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md

```diff
@@ -81,8 +77,8 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | `maxConcurrentHandlers` | N  |Defines the maximum number of concurrent message handlers. | `10`
 | `defaultMessageTimeToLiveInSec` | N  |Default message time to live. | `10`
 | `autoDeleteOnIdleInSec` | N  |Time in seconds to wait before auto deleting idle subscriptions. | `3600`
-| `maxReconnectionAttempts` | N  |Defines the maximum number of reconnect attempts. Default: `30` | `30`
-| `connectionRecoveryInSec` | N  |Time in seconds to wait between connection recovery attempts. Default: `2` | `2`
+| `minConnectionRecoveryInSec` | N | Minimum interval (in seconds) to wait before attempting to reconnect to Azure Service Bus in case of a connection failure. Default: `2` | `5`
```

Comment:
Did not see the service bug file here. Same comment for including this in the YAML example here as for SB queues comment above for both these entries.

Reply:
Also is this a breaking change or will the component still accept the original names?

Reply:
Made the change as requested.

This is a sort-of-but-not-really breaking change. The old option is ignored (but if it's still present, we show a warning in the logs and point to the docs) as it's not relevant anymore. The behavior it applied to was not desirable anyways. See conversation in https://github.com/dapr/components-contrib/pull/1783